### DISCUSSION TITLE
port2 overload

### DISCUSSION THREAD

**Discussion by bh4los** [jankae/LibreVNA#249] [Q&A]
I accidentally injected a 12dBm signal into the port2 for about 10 seconds, and the bottom right corner of the screen displayed ADS 
Overload. I am very worried that this will damage the port2. Later, I conducted measurements again and did not find issues. But I am still very worried. I'm not sure.

**Comment by jankae** [comment]
Not a problem at all. The +10 dBm maximum rating is very conservative. The first part that can actually get damaged is the mixer which has an absolute maximum input power rating of +20 dBm. And between the input and that mixer is the resistive bridge which will further attenuate the signal.
There is no way your VNA was damaged by a short +12 dBm signal.

**Reply by bh4los** [reply]
Thank you.I finally don't have to worry about it.
